src/dsymbol/tests.d

```diff
@@ -66,3 +69,77 @@ unittest
     q{auto b = [[[0]]];}.expectSymbolsAndTypes([["b", "*arr*", "*arr*", "*arr*", "int"]]);
 }
 
+static StringCache stringCache = void;
+static this()
+{
+    stringCache = StringCache(StringCache.defaultBucketCount);
+}
+
+const(Token)[] lex(string source)
+{
+    return lex(source, null);
+}
+
+const(Token)[] lex(string source, string filename)
+{
+    import dparse.lexer : getTokensForParser;
+    import std.string : representation;
+    LexerConfig config;
+    config.fileName = filename;
+    return getTokensForParser(source.dup.representation, config, &stringCache);
+}
+
+unittest
+{
+    import std.meta : AliasSeq;
+
+    auto tokens = lex(q{int a = 9;});
+    foreach(i, t;
+        AliasSeq!(tok!"int", tok!"identifier", tok!"=", tok!"intLiteral", tok!";"))
```

Comment:
You can use an array literal here, there is no need for the values to be available at compile time